internal/handlers/handlers.go

```diff
@@ -1 +1,51 @@
 package handlers
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func GetForm(w http.ResponseWriter, r *http.Request) {
+	resp, err := os.ReadFile("index.html")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	w.Write(resp)
+}
+
+func UploadForm(w http.ResponseWriter, r *http.Request) {
+	file, _, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
это ошибка пользователя
```suggestion
		http.Error(w, err.Error(), http.StatusBadRequest)
```